dk-installer.py

```diff
@@ -1363,8 +1384,8 @@ def create_compose_file(self, file, username, password, port, image, ssl_cert_fi
                     source: {ssl_key_file}
                     target: /dk/ssl/cert.key 
         """ if ssl_cert_file and ssl_key_file else ""
-        
-        file.write_text(
+        compose_file_path = get_docker_compose_path()
+        compose_file_path.write_text(
```

Comment:
if I understood the logic correct, this will crash on linux, because `get_docker_compose_path()` would return `None`